plugins/@grouparoo/cloudwatch/src/initializers/plugin.ts

```diff
@@ -0,0 +1,53 @@
+import { Initializer, loggers, env, ActionheroLogLevel } from "actionhero";
+import { URL } from "url";
+import * as winston from "winston";
+import WinstonCloudWatch from "winston-cloudwatch";
+
+const packageJSON = require("./../../package.json");
+
+export default class CloudwatchInitializer extends Initializer {
+  constructor() {
+    super();
+    this.name = packageJSON.name;
+    this.loadPriority = 1;
+  }
+
+  async initialize() {
+    if (!process.env.S3_ACCESS_KEY) return;
+    if (!process.env.S3_SECRET_ACCESS_KEY) return;
+    if (!process.env.S3_REGION) return;
+
+    // try to extract the name of the instance from the URL we are using
+    // we could alternatively use the ClusterName setting
+    let name = `grouparoo-${env}`;
+    if (process.env.WEB_URL) {
+      const parsed = new URL(process.env.WEB_URL);
+      name = parsed.host.split(":")[0].replace(/\./g, "-");
+    }
```

Comment:
@pedroslopez is there a better way to get the name of a running cloud instance?

Reply:
we could always inject a new env var `GROUPAROO_CLOUD_INSTANCE_NAME` or something similar